cloudinit/sources/DataSourceLXD.py

```diff
@@ -166,7 +167,7 @@ class DataSourceLXD(sources.DataSource):
     dsname = "LXD"
 
     _network_config: Union[Dict, str] = sources.UNSET
-    _crawled_metadata: Union[Dict, str] = sources.UNSET
+    _crawled_metadata: Optional[Union[Dict, str]] = sources.UNSET
```

Comment:
Hrm, LXD seems to be an outlier here with _crawled_metadata being set to sources.UNSET instead of None which the parent class attribute is initialized to. Should this actually by None instead of _UNSET to avoid potential issues with treating _UNSET as valid value != None? I see persist_instance_data treats != None as a viable value, even though we don't call the method until after the end of a successful get_data call.

Reply:
A change to LXD._crawled_metadata default value would involve a change to DataSourceLXD.network_config as well.

Reply:
Agreed. I wasn't sure if there was a good reason for having `UNSET` rather than `None`, but I'd rather not make that change here as it's unrelated to the changeset of this PR. If you know of no good reason, I can do it in a follow-up PR.